src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -123,13 +133,18 @@ protected static T Cast<T>(object value)
             where TDeclarationNode2 : SyntaxNode
             where TDeclarationNode3 : SyntaxNode
         {
+            if (destination == null)
```

Comment:
Can you please file a bug to track cleaning up entire CodeGen service to handle null arguments for all public entry points? I remember seeing a few more elsewhere.